ios/app/src/ViewController.m

```diff
@@ -96,4 +96,10 @@ - (void)conferenceWillJoin:(NSDictionary *)data {
     [self _onJitsiMeetViewDelegateEvent:@"CONFERENCE_WILL_JOIN" withData:data];
 }
 
+#if 0
```

Comment:
how does it work `#if 0` ?

Reply:
it doesn't :-) it's there to make testing easier. just change it to 1 and one should see the change in behavior.

Reply:
then I don't get it, why is this code even here ?

Reply:
So we don't have to write it, eevery time it needs testing.